mozregression/cli.py

```diff
@@ -248,6 +248,12 @@ def create_parser(defaults):
         help="application name. Default: %(default)s.",
     )
 
+    parser.add_argument(
+        "--lang",
+        metavar="[ar|es-ES|he|ja|zh-CN|...]",
+        help=("build language. Only valid when app is " "firefox-l10n."),
```

Comment:
```suggestion
        help=("build language. Only valid when app is firefox-l10n."),
```

Reply:
Ah, looks like an artefact of rebasing + auto-formatting, good catch thanks.